nbbuild/default.xml

```diff
@@ -134,8 +134,10 @@
     <!-- Writing the module's cluster to property file -->
     <propertyfile file="${moduleCluster.file}">
       <entry  key="${module.name}.dir" value="$${netbeans.dest.dir}/${cluster.dir}"/>
+      <entry  key="${module.id}.dir" value="$${netbeans.dest.dir}/${cluster.dir}"/>
     </propertyfile>
     <property name="${module.name}.dir" value="${netbeans.dest.dir}/${cluster.dir}"/>
+    <property name="${module.id}.dir" value="${netbeans.dest.dir}/${cluster.dir}"/>
```

Comment:
Define both `api.io.dir` as well as `platform/api.io.dir` once the layout #536 moves `api.io` module into `platform` subdirectory.